### ISSUE TITLE
gdbm_fetch: Item not found at /usr/bin/obmenu-generator line 492

### ISSUE THREAD

**Issue by miramir** [trizen/obmenu-generator#40]
distr: gentoo
Clean setup.
```
$ obmenu-generator -i
gdbm_fetch: Item not found at /usr/bin/obmenu-generator line 492.
```

**Comment by trizen** [comment]
Hmm.. I think `gdbm` is built to throw an error when an item is not found. Try to rebuild `gdbm` with the default options.

**Comment by snekiepete** [comment]
getting the same error even after gdbm rebuild

**Comment by trizen** [comment]
Please check the latest commit (https://github.com/trizen/obmenu-generator/commit/6901b02385e208fd158a6189651e2dbe038f8c70) to see if it indeed fixes the issue. Thanks.

**Comment by catthou** [comment]
Latest commit fixed the issue for me on gentoo.

**Comment by trizen** [comment]
Just released version 0.90 with this fix: https://github.com/trizen/obmenu-generator/releases/tag/0.90

Thanks for confirmation!
